lib/northbound.c

```diff
@@ -1213,6 +1221,9 @@ static int nb_callback_create(struct nb_context *context,
 	bool unexpected_error = false;
 	int ret;
 
+	if (CHECK_FLAG(nb_node->flags, F_NB_NODE_IGNORE_CBS))
+		return 0;
```

Comment:
This check is redudant, the caller has already done it. This should just be an assert.

Reply:
Ok. Will make it assert instead.